### PR TITLE
Add widgets for custom charts and jupyterlite notebooks

### PR DESCRIPTION
**Pull request by alexmojaki**

### CODE REVIEW

File: chart/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@gristlabs/widget-chart",
+  "description": "Widget for more customizable charts with plotly",
+  "homePage": "https://github.com/gristlabs/custom-charts-widget",
+  "version": "0.0.1",
+  "grist": [
+    {
+      "name": "Chart",
```
Comment (paulfitz) [inline chart/package.json:8]: A more specific name would be better.

Comment (paulfitz) [inline chart/package.json:8]: A more specific name would be better.

File: jupyterlite/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "@gristlabs/widget-jupyterlite",
+  "description": "Widget for a JupyterLite notebook",
+  "homePage": "https://github.com/gristlabs/jupyterlite-widget",
+  "version": "0.0.1",
+  "grist": [
+    {
+      "name": "Chart",
```
Comment (paulfitz) [inline jupyterlite/package.json:8]: The name is out of sync

**Review by paulfitz — CHANGES_REQUESTED** [review]
This approach worries me, and changes how the widget collection works in ways that could be hard to step back from. I'd rather have someone look seriously at alternatives before committing. Placing this on hold, since use of these widgets by URL is enough for now.

**Comment by berhalak** [comment]
How about small mental model shift. Lets treat this github repository as a package repository (like npm), so it contains all prebuilt artifacts (js files). And in the chart and jupiter repository, you would just add a github action that will build your project and create a pull request for this one, with everything built already and configured properly?
So you will use it (this repo) as a hosting provider for your widgets, and all the source code and tests and all the machinery will still sit in your repository.

**Review by paulfitz — APPROVED** [review]
Thanks @alexmojaki !

**Pull request merged**